cypress/integration/types/constants.ts

```diff
@@ -14,6 +14,7 @@ export const jobfunctions = "Job functions";
 export const jobfunction = "Job function";
 export const memberCount = "Member count";
 export const name = "Name";
+export const members = "Member(s)";
```

Comment:
This has been updated in new UI. The new word is Member count, which is already present in the file.